docs/ROADMAP.md

```diff
@@ -7,38 +7,56 @@ category: Roadmap
 
 This document defines the roadmap for TiKV development.
 
-## Raft
-- [x] Region Merge - Merge small Regions together to reduce overhead
-- [x] Local Read Thread - Process read requests in a local read thread
-- [x] Split Region in Batch - Speed up Region split for large Regions
-- [x] Raft Learner - Support Raft learner to smooth the configuration change process
-- [x] Raft Pre-vote - Support Raft pre-vote to avoid unnecessary leader election on network isolation
-- [ ] Joint Consensus - Change multi members safely.
-- [ ] Multi-thread Raftstore - Process Region Raft logic in multiple threads
-- [ ] Multi-thread apply pool - Apply Region Raft committed entries in multiple threads
-
 ## Engine
-- [ ] Titan - Separate large key-values from LSM-Tree
+- [ ] Titan - Use the separated key-value engine in production
 - [ ] Pluggable Engine Interface - Clean up the engine wrapper code and provide more extendibility
 
-## Storage
-- [ ] Flow Control - Do flow control in scheduler to avoid write stall in advance
-
-## Transaction
-- [x] Optimize transaction conflicts
-- [ ] Distributed GC - Distribute MVCC garbage collection control to TiKV
+## Raft
+- [ ] Joint Consensus - Change multi members safely
```

Comment:
There should be a new line after the title to keep consistent with other titles.